examples/step-12/step-12.cc

```diff
@@ -209,20 +208,15 @@ namespace Step12
     // fact other arguments already bound.
     static void integrate_cell_term(DoFInfo &dinfo, CellInfo &info);
     static void integrate_boundary_term(DoFInfo &dinfo, CellInfo &info);
-    static void integrate_face_term(DoFInfo & dinfo1,
-                                    DoFInfo & dinfo2,
-                                    CellInfo &info1,
-                                    CellInfo &info2);
+    static void
+    integrate_face_term(DoFInfo &dinfo1, DoFInfo &dinfo2, CellInfo &info1, CellInfo &info2);
```

Comment:
:-(

Reply:
C'Mon. This here is a matter of taste. If you really don't like it we can penalize the break after `void` more to discourage this outcome.

Reply:
Yes, matter of taste. (Though we've codified this also in our coding conventions.)

Reply:
#6696 fixes this.